src/ui/selectors.ts

```diff
@@ -1,15 +1,6 @@
 import { ReduxState } from "types";
 
 // ----- Exported functions
-
```

Comment:
Removing this comment for consistency as `isConfirmationModalActive ` was added without comment. It's also pretty self-explanatory.